pkg/systemd/services/services.jsx

```diff
@@ -993,7 +990,8 @@ const ServicesPageFilters = ({
         <Toolbar data-loading={loadingUnits}
                  clearAllFilters={onClearAllFilters}
                  className="pf-m-sticky-top ct-compact services-toolbar"
-                 id="services-toolbar">
+                 id="services-toolbar"
+                 numberOfFiltersText={n => cockpit.format("$0 filters applied")}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17961-20221129-122228-00a35c6d-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)